.github/workflows/main.yml

```diff
@@ -41,17 +39,6 @@ jobs:
           repository: WeMoveEU/org.project60.sepa
           ref: 1.5.0.patched
           path: drupal/sites/all/modules/civicrm/ext/sepa
-      - uses: sudosubin/git-clone-action@v1.0.1
-        with:
-          platform: lab.civicrm.org
-          repository: extensions/mjwshared
-          ref: master
-          path: drupal/sites/all/modules/civicrm/ext/mjwshared
-      - uses: actions/checkout@v2
```

Comment:
Have you checked whether the event processing code works without Stripe extension? I think I added the dependency because it's required (e.g. to declare the Stripe payment processor).

Reply:
Yah we'll need to keep it for migrating subscriptions from Commit change for now too. I think I was wrong about the conflict. We'll definitely need to disable the webhook in Stripe.